### ISSUE TITLE
Use Zalenium to run the integration tests for tailor on multiple environments

### ISSUE THREAD

**Issue by addityasingh** [zalando/tailor#156]
From docs:
> Zalenium
This is a Selenium Grid extension to scale your local grid dynamically with docker containers. It uses docker-selenium to run your tests in Firefox and Chrome locally, and when you need a different browser, your tests can get redirected to a cloud testing provider (Sauce Labs, BrowserStack, TestingBot).

https://github.com/zalando/zalenium

**Comment by vigneshshanmugam** [comment]
Not sure we need this functionality for tailor, we already have saucelabs integration which works without any issues. But if you have experience using Zalenium and want to contribute, Feel free to send PR.

**Comment by vigneshshanmugam** [comment]
Closing this since its not a priority and current solution works for most cases.

**Comment by hjacobs** [comment]
You could try out Continuous Delivery Platform + Zalenium --- @diemol is probably willing to help :smile:

**Comment by vigneshshanmugam** [comment]
@hjacobs Thanks! Totally in if someone is willing to help move from the current setup.

**Comment by diemol** [comment]
I was checking around, I see you already have a Sauce Labs account, I guess an open source one, right?

If that's correct, I am not sure if it makes sense to integrate Zalenium. Nevertheless, I was checking the project and I couldn't figure out quickly where to make changes. Perhaps it would be helpful to add this to the contributing guidelines?

**Comment by vigneshshanmugam** [comment]
@diemol Yes its a open source one. The integration test reside here https://github.com/zalando/tailor/blob/master/examples/multiple-fragments-with-custom-amd/test.js

**Comment by diemol** [comment]
I'll assign this to me, I'll get back to you in case I have questions. Thanks.

**Comment by diemol** [comment]
Hi @vigneshshanmugam,

I checked this again, and as much as I'd like for this project to use Zalenium, I don't think it makes sense. I am more an advocate of using the right tool than anything else. 

If tailor uses Zalenium, then the tests running with Chrome would use Zalenium directly in the Travis job, video gets recorded but this would be lost since there is not at straightforward way to store test results. When you run with IE, those ones would go to SauceLabs and the videos will be stored there.

If tailor keeps only using SauceLabs, then all the videos are already stored there. Which fits well in this case.

In short, adding Zalenium would put one layer more into the flow without bringing a lot of value.

In my opinion, this could be closed.

**Comment by vigneshshanmugam** [comment]
@diemol Thanks for your detailed insights. Closing it as per your suggestion.
